profile.html

```diff
@@ -103,54 +103,54 @@ <h5 class="card-title">Duckie Duckerson</h5>
                     </div>
                 </div>
             </div>
-            <div class="friends card col-4 mb-3 pb-3">
-                <h3 class="mt-2 ms-3">My Friends</h3>
-                <div class="profile-header m-2">
+            <div class="friends row card col-9 col-md-4 mb-3 pb-3">
+                <h3 class="mt-2 ps-3">My Friends</h3>
+                <div class="profile-header py-2">
                     <div class="row">
                         <div class="friends-img col-4">
                             <div class="col-md-12">
                                 <img src="/images/unicorn-duck.jpg" class="img-fluid rounded-circle" alt="User profile pic">
                             </div>
                         </div>
-                        <div class="col-8">
-                            <h5 class="m-0 mb-1">Queen Tuttifrutti</h5>
-                            <span class="text-muted">Quack Queen</span>
-                            <div class="row row-cols-2 mt-2">
-                                <button class="btn btn-primary btn-round ms-2">Follow</button>
+                        <div class="col-8 row">
+                            <h5 class="m-0 p-0 mb-1">Queen Tuttifrutti</h5>
```

Comment:
```suggestion
                            <div class="col-12"><h5 class="m-0 p-0 mb-1">Queen Tuttifrutti</h5></div>
```
Child of a row should be a column